### DISCUSSION TITLE
Add a push workflow as a check to a PR

### DISCUSSION THREAD

**Discussion by cclaes-dsi** [community/community#80889] [Actions] [answered]
### Select Topic Area

Question

### Body

We currently have 2 Actions workflows. A workflow that runs on every push to `development/main/feature` branches and a workflow that runs when a PR is opened.

Is it possible to add the push workflow as a check to the PR as well? 

Adding `on: pull_request`to the push workflow isn't really an option since our unit tests check the branch name etc and in the PR the branch names get changed. That's why we have 2 separate ones.

**Comment by cclaes-dsi** [comment, marked as answer]
Our branch pattern in the push workflow was incorrect, that's why it wasn't checking those.

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
